README.md

```diff
@@ -1,44 +1,71 @@
+# GSP [![IRC](https://img.shields.io/badge/kubernetes-v1.12.6-0099ef.svg)]() <img align="right" src="./docs/assets/gsp.png" alt="gsp" width="30%" height="whatever">
 
-# What is the GDS Supported Platform?
+GSP (GDS Supported Platform) is a Kubernetes distribution maintained by the [Govenement Digital Service](https://www.gov.uk/government/organisations/government-digital-service) created to meet the common needs of running digital services in production.
```

Comment:
@chrisfarms I think we need to be more explicit about who this is for. i.e. GDS Service Teams

Reply:
Our needs have def come from GDS services... but it _could_ be used by others who want to follow the same path 🤷‍♂